ocaml/doc/README.md

```diff
@@ -1,13 +1,14 @@
 # A note on generating locally the API reference
 
-Run `make doc`. The output is in `_build/ocaml/doc/markdown`.
+Run `make doc`. The output is in `_build/ocaml/doc`.
```

Comment:
I'll extend the README, but I'm not sure it's a good idea to expose these targets in the Makefile, because it gives the impression it should run out of the box, but it doesn't because pandoc and latex are not in the dependencies.